addons/format_mp3.c

```diff
@@ -28,6 +32,7 @@
  */
 
 /*** MODULEINFO
+	<depend>lame</depend>
```

Comment:
Assuming you stick with this approach (mpg123 & lame in the same file) I believe this should be:

    <use type="external">lame</use>